go/libkb/httpsrv.go

```diff
@@ -110,10 +110,9 @@ func (h *HTTPSrv) Start() (err error) {
 		Addr:    address,
 		Handler: h.ServeMux,
 	}
+	// Set active to true before returning.
+	h.active = true
```

Comment:
Where is this used?

Reply:
In `Active()`, it uses this field to determine if the server is up.